packages/visualizations/src/components/Chart/scales.ts

```diff
@@ -81,6 +81,9 @@ export default function buildScales(options: ChartOptions): ChartJsChartOptions[
     // X Axis
     if (options.axis?.x) {
         scales.x = {
+            ...(options.axis.x.type === 'linear'
+                ? { beginAtZero: defaultValue(options?.axis?.x?.beginAtZero, false) }
+                : {}),
```

Comment:
```suggestion
            ...(options.axis.x.type === 'linear' && { beginAtZero: defaultValue(options?.axis?.x?.beginAtZero, false) }),
```

Maybe a more compact way to write it, should have the same output ?